app/index.html

```diff
@@ -60,116 +60,104 @@
       margin: auto;
       width:100%;
     }
-.spinner-wrapper{
-  width:100%;
-  height:100vh;
-  position:fixed;
-  overflow:hidden;
-  top:0;left:0;
-  z-index:999;
-  display:none;
-}
-body.waiting .spinner-wrapper{
-  display:block;
-}
-.spinner-wrapper--bg{
-  width:100%;
-  height:100vh;
-  position:absolute;
-  top:0;left:0;
-  overflow:hidden;
-z-index:0;
-background-color:#7ccff4;
-opacity:0.4;
-}
-
-.spinner-wrapper--container{
-  position:absolute;
-  top:0;left:0;right:0;bottom:0;
-  margin:auto;
-  width:80px;
-  height:80px;
-  z-index:1;
-}
+    .spinner-wrapper {
+      width:100%;
+      height:100vh;
+      position:fixed;
+      overflow:hidden;
+      top:0;left:0;
+      z-index:999;
+      display:none;
+    }
+    body.waiting .spinner-wrapper{
+      display:block;
+    }
+    .spinner-wrapper--bg{
+      width:100%;
+      height:100vh;
+      position:absolute;
+      top:0;left:0;
+      overflow:hidden;
+    z-index:0;
```

Comment:
These three lines 2 spaces

Reply:
You're right... I'll fix that in the next PR.
Thanks for the review